gpii/node_modules/settingsHandlers/src/settingsHandlerUtilities.js

```diff
@@ -74,24 +74,39 @@ gpii.settingsHandlers.transformPayload = function (payload, handler) {
     });
 };
 
+/** Transform the settings for one solution's worth of settings by a supplied transform.
+ * @param oneSolution {Object} One solution's worth of settings, with a member named `settings` containing a free hash of the settings indexed by settingsHandlerBlock name
+ * @param handler {Function} A transformer whose signature is the same as the 2nd argument to gpii.settingsHandlers.transformPayload
+*/
+
+gpii.settingsHandlers.transformOneSolutionSettings = function (element, handler, path) {
```

Comment:
The parameters in the function documentation don't match the parameters in the function definition: `oneSolution` and `handler` vs `element`, `handler`, and `path`.